src/mpid/ch4/include/mpidpre.h

```diff
@@ -230,6 +250,14 @@ typedef struct MPIDI_CH4U_win_info_args_t {
     int alloc_shared_noncontig;
     MPIDI_CH4U_win_info_accumulate_ops accumulate_ops;
 
+    /* hints to tradeoff atomicity support */
+    int which_accumulate_ops;   /* Arbitrary combination of {max,min,sum...}.
```

Comment:
My concern is that, the value can be multiple ops, which is not covered by the value range of enum. Maybe define the op values as macro instead of enum is more straightforward. Then the which_accumulate_ops type can be `unsigned int` or `uint32_t`

Reply:
I improved the comment here to make it clear. And changed to uint32_t.

Reply:
Oh you're right. Disregard my comment then.